sim/warlock/demonology/hand_of_guldan.go

```diff
@@ -0,0 +1,74 @@
+package demonology
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/warlock"
+)
+
+func (demonology *DemonologyWarlock) CurseOfGuldanDebuffAura(target *core.Unit) *core.Aura {
+	return target.GetOrRegisterAura(core.Aura{
+		Label:    "CurseOfGuldan-" + demonology.Label,
+		ActionID: core.ActionID{SpellID: 86000},
+		Duration: time.Second * 15,
+		OnGain: func(aura *core.Aura, sim *core.Simulation) {
+			//TODO: Implement Crit rating for pet vs this target only
+		},
+		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+			//TODO: Implement Crit rating for pet vs this target only
+		},
+	})
+}
+
+// TODO: Since we are assuming hand of guldan will hit all targets can we just give the pets 10% crit for the duration?
```

Comment:
Looking at the spell it seems it has a Hit check so we cant assume its hitting all targets